native-image-tests/src/main/kotlin/okhttp3/GenerateClassList.kt

```diff
@@ -0,0 +1,15 @@
+package okhttp3
+
+import org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor
+import java.io.File
+
+/**
+ * Run periodically to refresh the known set of working tests.
+ *
+ * TODO use filtering to allow skipping acceptable problem tests
```

Comment:
Better to integrate with PlatformTest, like Platform.isGraalvm() ?

Reply:
Yep, probably in a follow up. I'd still like to use tags. Go full junit5 native on this.